rasa/nlu/extractors/lookup_entity_extractor.py

```diff
@@ -0,0 +1,129 @@
+import logging
+import os
+from typing import Any, Dict, List, Optional, Text
+
+import rasa.utils.common as common_utils
+from rasa.constants import DOCS_URL_COMPONENTS
+from rasa.nlu import utils
+from rasa.nlu.components import Component
+from rasa.nlu.config import RasaNLUModelConfig
+from rasa.nlu.constants import ENTITIES
+from rasa.nlu.extractors.extractor import EntityExtractor
+from rasa.nlu.model import Metadata
+from rasa.nlu.training_data import Message
+
+logger = logging.getLogger(__name__)
+
+
+class LookupEntityExtractor(EntityExtractor):
+    """
+    Searches for entities in the user's message from a list of examples.\n
+    Required Parameters: \n
+    @entities -> list \n
+    @files_path -> list
+    """
+
+    defaults = {
+        "entities": None,
+        "files_path": None
```

Comment:
Can you please add comments what those values are for? Thanks.

Reply:
If I understand it correctly `entities` and `files_path` are both lists. So the first file path lists all values of the first entity type listed in `entities`. Maybe it would be simple to store this as a dict. For example
```
- name: LookupEntityExtractor
   lookup:
     city: /some/path/city.txt
     person: /some/other/path/person.txt
```
What do you think?

Reply:
If you store them as dict, it would also simplify the checks in `__init__` quite a bit.

Reply:
@tabergma thanks for the review, I will make those changes